drupal/scripts/init_drupal.sh

```diff
@@ -58,6 +58,9 @@ LANG_INFO=$(drush language-info --field=language)
 drush language:default -y "fi"
 
 # enable base theme
+drush pm:enable -y jquery_ui
+drush pm:enable -y jquery_ui_draggable
+drush pm:enable -y jquery_ui_resizable
```

Comment:
These should check if the they are already enabled for a slight speed up, e.g. https://github.com/vrk-kpa/opendata/blob/50f1d1cfa7d643b5cf4edaf0aacd4d0ea2ec413c/drupal/scripts/init_drupal.sh#L46-L50

And there should be some comment why they are there, so that we remember to remove them when they aren't needed anymore.